BedrockServer.cpp

```diff
@@ -1004,13 +1004,45 @@ BedrockServer::BedrockServer(const SData& args)
     // Enable the requested plugins, and update our version string if required.
     list<string> pluginNameList = SParseList(args["-plugins"]);
     vector<string> versions = {_version};
+
+    // A map of plugins and what secure data entries they need.
+    map<BedrockPlugin*, list<string>> pluginSecureDataMap;
     for (string& pluginName : pluginNameList) {
         BedrockPlugin* plugin = registeredPluginMap[SToLower(pluginName)];
         if (!plugin) {
             SERROR("Cannot find plugin '" << pluginName << "', aborting.");
         }
-        plugin->initialize(args, *this);
+
+        // Does this plugin need secured data? Plugin secure data is a method of
+        // storing sensitive data that you wouldn't want to commit into your code
+        // base (API keys, encryption keys, etc). Bedrock will allow a plugin to
+        // define a set of system file locations to load secure data from it (if
+        // the plugin requires any). These locations can be prefaced with `socket:`,
+        // in which case Bedrock will read the secure data from a socket (specified
```

Comment:
What is prefixed with `socket`? Any random string, or the location of the socket to read from? If the second, what do `-secureDataHost` and the default do?